src/app/services/admin-patient.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
+import { ServerApiService } from './server-api.service';
+import { Observable } from 'rxjs';
+import { Response } from '../../../api-objects/GenericResponse';
+import { AdminPatientResponse, ResetPasswordResponse, UserListItem } from '../../../api-objects/AdminPatient';
+import { PatientResponse } from '../../../api-objects/patientResponse';
+
+@Injectable()
+export class AdminPatientService {
+  constructor(private serverApiService: ServerApiService) { }
+
+  public createPatient(patientId : string, deceased : boolean, patientNotes : string) : Observable<Response<PatientResponse>> {
+    return this.serverApiService.post('patient', {'patientId': patientId, 'deceased': deceased, 'patientNotes': patientNotes});
+  }
+
+  public getPatients() : Observable<Response<PatientResponse[]>> {
+    return this.serverApiService.get<PatientResponse[]>('patient');
+  }
+
+  public getPatient(patientId : number) : Observable<Response<PatientResponse[]>> {
+    return this.serverApiService.get<PatientResponse[]>('patient/' + patientId);
+  }
+
+  public deletePatient(patientId : number) : Observable<Response<any>> {
+    return this.serverApiService.delete<any>('patient/' + patientId);
+  }
+
+  public updateNotes(patientId : number, notes : string) : Observable<Response<any>> {
+    return this.serverApiService.post<any>('patient/' + patientId + '/notes', {notes});
+  }
```

Comment:
Same as with deletePatient.

Reply:
The endpoint returns: { 'notes': string }
So this should as well.  Maybe a later ticket could create a class/interface in api-objects.

Reply:
This shouldn't return the note data. It should also be a null return (Signifying 'SUCCESS')

Reply:
I was under the impression the service returns whatever the api gives back? Is there a reason the update notes for patient api returns a copy of the notes?

Reply:
No. Not that I'm aware of.